src/zero/module/copyfile.clj

```diff
@@ -5,7 +5,7 @@
             [zero.module.all :as all]
```

Comment:
No functional changes to this file. Revert?

Reply:
Not now, please.